app/lib/server/functions/addOAuthService.js

```diff
@@ -0,0 +1,37 @@
+/* eslint no-multi-spaces: 0 */
+/* eslint comma-spacing: 0 */
+import s from 'underscore.string';
+
+import { settings } from '../../../settings';
+
+export function addOAuthService(name, values) {
```

Comment:
```suggestion
export function addOAuthService(name, values = {}) {
```

Reply:
The addOAuthService method doesn't send any data to the function so a default value is needed here.

Reply:
Good catch!  I thought about that but forgot to add it :see_no_evil: